extensions/spotify/init.lua

```diff
@@ -125,4 +153,54 @@ function spotify.getCurrentTrack()
     return tell('name of the current track')
 end
 
+--- hs.spotify.getPlayerState()
+--- Function
+--- Gets the current state of Spotify
+---
+--- Parameters:
+---  * None
+---
+--- Returns:
+---  * A string containing one of hs.spotify.state_stopped, hs.spotify.state_paused, hs.spotify.state_playing
+function spotify.getPlayerState()
+   return tell('get player state')
+end
+
+--- hs.spotify.isRunning()
+--- Function
+--- Returns whether Spotify is currently open. Most other functions in hs.spotify will automatically start the application, so this function can be used to guard against that.
+---
+--- Parameters:
+---  * None
+---
+--- Returns:
+---  * A boolean value indicating whether the Spotify application is running.
+function spotify.isRunning()
+   return hs.application.get("Spotify") ~= nil
```

Comment:
Thanks, I was wondering about this, and also about calling functions within the same module (e.g. the call to `hs.spotify.isRunning()` from `isPlaying()`. Is the way I did it correct?